packages/orbit-components/src/Skeleton/Skeleton.stories.tsx

```diff
@@ -93,6 +100,40 @@ export const Presets = () => {
   return <Skeleton preset={presets} />;
 };
 
+export const VisualTest = () => {
```

Comment:
One last thing. I think we shoud set `animte=false` explicitly, so it won't fail on visual regression 🤔